### PR TITLE
KS-SK1183-Update-DNSSEC-for-DomainDNS-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done except EN subs duplication from ENGB.

Need proof for DE, ENGB, latin and pl versions.

### CODE REVIEW

File: pages/web/domains/dns_dnssec/guide.en-gb.md [hunk 1]
```diff
@@ -1,65 +1,83 @@
...
+
+You can also refer to our guides on [OVHcloud DNS servers](/pages/web/domains/dns_server_general_information/) and on [editing an OVHcloud DNS zone](/pages/web/domains/dns_zone_edit/) if you would like more information on these topics.
 
 ## Requirements
 
-- a domain name registered with OVHcloud
+- a domain name registered with OVHcloud.
 - The domain name concerned must have an extension compatible with DNSSEC.
-- access to the `Web Cloud`{.action} section of the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external}
+- You must be logged in to your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external}, part `Web Cloud`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.en-gb.md]:
```suggestion
- You must be logged in to your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external} and access the `Web Cloud`{.action} section.
```

File: pages/web/domains/dns_dnssec/guide.en-gb.md [hunk 2]
```diff
@@ -1,65 +1,83 @@
...
+>
+> If the DNS server names end with *ovh.net*, *ovh.ca* or *anycast.me*, your domain name will use the OVHcloud DNS servers.
 >
 
-### Step 1: Access the domain name management section.
+### Step 1: Access the domain name management <a name="step1"></a>
 
-To start with, log in to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external}, and go to the `Web Cloud`{.action} section. Click `Domains`{.action}, then choose the domain name concerned from the list.
+To activate the **DNSSEC** solution for your domain name, log in to your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external} then go to the `Web Cloud`{.action} section. In the left-hand column, click `Domain names`{.action} , then choose the domain name concerned from the list.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.en-gb.md]:
```suggestion
To activate the **DNSSEC** solution for your domain name, log in to your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external} then go to the `Web Cloud`{.action} section. In the left-hand column, click `Domain names`{.action}, then choose the domain name concerned from the list.
```

File: pages/web/domains/dns_dnssec/guide.en-gb.md [hunk 3]
```diff
@@ -1,65 +1,83 @@
...
-### Step 3: Wait during the activation or deactivation.
-
-Once you have decided to activate or deactivate DNSSEC on your domain name, you will need to wait a maximum of 24 hours for the change to take effect. 
+> [!primary]
+>
+> The activation/deactivation of **DNSSEC** takes **24** hours to be effective.
+>
+> Furthermore, if you later want to change the DNS servers associated with your domain name, the DNS servers will not be modified on the OVHcloud side until the **DNSSEC** has been disabled. After this, an additional period of **24** to **48** hours will be required for the DNS propagation of the modification.
+>
+> In total, the modification of a domain name’s DNS servers with the **DNSSEC** active solution will be fully effective after **48** at **72** hours.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.en-gb.md]:
```suggestion
> In total, the modification of a domain name’s DNS servers with the **DNSSEC** active solution will be fully effective after **48** to **72** hours.
```

File: pages/web/domains/dns_dnssec/guide.es-es.md [hunk 4]
```diff
@@ -1,64 +1,87 @@
...
+En algunos casos, los flujos de datos que transitan por los servidores DNS pueden ser utilizados por personas malintencionadas.
+Para ello, estas personas envenenan la caché de los servidores DNS con la configuración DNS que desean aplicar a su dominio: es lo que se llama "cache poisoning".
+De este modo, pueden redirigir los flujos entrantes de su dominio hacia sus propios sitios web y hacia sus propias direcciones de correo.
 
-**Esta guía explica cómo activar DNSSEC en un dominio para protegerlo contra el *DNS cache poisoning*.** Para más información, le recomendamos que consulte nuestra página relativa a [DNSSEC](https://www.ovhcloud.com/es-es/domains/dnssec/){.external}.
+El **D**omain **N**ame **S**ystem **SEC**urity extensions (**DNSSEC**), permite proteger la configuración DNS de su nombre de dominio contra el "cache poisoning" verificando y autentificando las respuestas DNS.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.es-es.md]:
```suggestion
El **D**omain **N**ame **S**ystem **SEC**urity extensions (**DNSSEC**) permite proteger la configuración DNS de su nombre de dominio contra el "cache poisoning" verificando y autentificando las respuestas DNS.
```

File: pages/web/domains/dns_dnssec/guide.es-es.md [hunk 5]
```diff
@@ -1,64 +1,87 @@
...
+
+Si desea más información sobre estos temas, no dude en consultar nuestras guías sobre [los servidores DNS OVHcloud](/pages/web/domains/dns_server_general_information/) y sobre la [edición de una zona DNS OVHcloud](/pages/web/domains/dns_zone_edit/).
 
 ## Requisitos
 
-- Tener un dominio registrado en OVHcloud.
+- Tener un dominio registrado con OVHcloud.
 - El dominio debe tener una extensión compatible con DNSSEC.
-- Estar conectado al [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, en la sección `Web Cloud`{.action}.
+- Estar conectado a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, parte `Web Cloud`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.es-es.md]:
```suggestion
- Estar conectado a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, en la parte `Web Cloud`{.action}.
```

File: pages/web/domains/dns_dnssec/guide.es-es.md [hunk 6]
```diff
@@ -1,64 +1,87 @@
...
-Haga clic en la pestaña `Registros DS`{.action}, a continuación, haga clic en el botón Editar de la derecha y, por último, en el botón `+`{.action}.
-Rellene los 4 campos "Key tag", "Flag", "Algorithm", "Public key (codificado en base 64)" con los datos que le ofrece su actual proveedor.
+- **Su dominio no utiliza los servidores DNS de OVHcloud** : Contacte con el proveedor que gestione la configuración DNS del dominio para pedirle su configuración. Conéctese a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external} y acceda a la sección `Web Cloud`{.action}. En la columna izquierda, haga clic en `Dominios`{.action} y seleccione el dominio correspondiente en la lista.</br>
+Seleccione la pestaña `Registros DS`{.action}, haga clic en el botón `Editar`{.action} a la derecha y, seguidamente, en el botón `+`{.action} que aparece.</br>.
+Ya puede introducir los 4 campos "Key Tag", "Flag", "Algoritmo", "Clave pública (codificada en base 64)" con los datos comunicados por su actual proveedor.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.es-es.md:41]: Are the 4 fields translated in the manager?

File: pages/web/domains/dns_dnssec/guide.it-it.md [hunk 7]
```diff
@@ -1,66 +1,87 @@
----
-title: 'Rendere sicuro un dominio con DNSSEC'
-excerpt: 'Come proteggere un dominio dal Cache Poisoning attivando DNSSEC'
-legacy_guide_number: g609
-updated: 2023-04-26
----
+---
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_dnssec/guide.it-it.md]: Looks like something is wrong here. Not the same coloration from GitHub compared to other versions.
Same thing for pt-pt.
Please double-check

File: pages/web/domains/dns_dnssec/guide.es-es.md (hunk 6 above)
Comment (Kazarus4) [reply pages/web/domains/dns_dnssec/guide.es-es.md:41]: Yes, Itranslated with a manager v6 opened

File: pages/web/domains/dns_dnssec/guide.it-it.md (hunk 7 above)
Comment (Kazarus4) [reply pages/web/domains/dns_dnssec/guide.it-it.md]: all good on vscode

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**